modules/distr-tracing-tempo-install-cli.adoc

```diff
@@ -0,0 +1,235 @@
+[id="distr-tracing-tempo-install-cli_{context}"]
+= Installing the {TempoShortName} by using the CLI
+
+You can install the {TempoShortName} from the command line.
+
+.Prerequisites
+
+* An active {oc-first} session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your {oc-first} version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username>
+----
+====
+
+* You are using a supported provider of object storage: link:https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation], link:https://min.io/[MinIO], link:https://aws.amazon.com/s3/[Amazon S3], link:https://azure.microsoft.com/en-us/products/storage/blobs/[Azure Blob Storage], link:https://cloud.google.com/storage/[Google Cloud Storage].
+
+.Procedure
+
+. Install the {TempoOperator}:
+
+.. Create a project for the {TempoOperator} by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  labels:
+    kubernetes.io/metadata.name: openshift-tempo-operator
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-tempo-operator
+EOF
+----
+
+.. Create an operator group by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-tempo-operator
+  namespace: openshift-tempo-operator
+spec:
+  upgradeStrategy: Default
+EOF
+----
+
+.. Create a subscription by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: tempo-product
+  namespace: openshift-tempo-operator
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: tempo-product
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+.. Check the operator status by running the following command:
++
+[source,terminal]
+----
+$ oc get csv -n openshift-tempo-operator
+----
+
+. Create a secret for your object storage bucket by running one of the following commands:
+
+** To create a secret from a YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <secret_file>.yaml
+----
+
+** To create a secret from standard input:
++
+[source,terminal]
+----
+$ oc apply -f - << EOF
+<object_storage_secret>
+EOF
+----
++
+--
+include::snippets/distr-tracing-tempo-required-secret-parameters.adoc[]
+--
++
+--
+include::snippets/distr-tracing-tempo-secret-example.adoc[]
+--
+
+. Create a project of your choice for the *TempoStack* instance that you will create in the next step:
+
+** To create a project from standard input without metadata:
++
+[source,terminal]
+----
+$ oc new-project <project_of_tempostack_instance>
+----
+
+** To create a project from standard input with metadata:
+[source,terminal]
```

Comment:
Apologies, I was checking the rendering myself earlier today and clearly missed this rendering error.
I've amended the PR just now.
Waiting for the preview to build.
Will ping you when the preview is ready so that you can verify it yourself.